pkg/controller/autoimport/autoimport_controller.go

```diff
@@ -78,100 +90,86 @@ func (r *ReconcileAutoImport) Reconcile(ctx context.Context, request reconcile.R
 		return reconcile.Result{}, err
 	}
 
-	reqLogger.Info("Reconciling auto import secret")
+	reqLogger.V(5).Info("Reconciling auto import secret")
 
-	importSecretName := fmt.Sprintf("%s-%s", managedClusterName, constants.ImportSecretNameSuffix)
-	importSecret, err := r.informerHolder.ImportSecretLister.Secrets(managedClusterName).Get(importSecretName)
-	if errors.IsNotFound(err) {
-		// there is no import secret, do nothing
-		return reconcile.Result{}, nil
-	}
-	if err != nil {
-		return reconcile.Result{}, err
+	lastRetry := 0
+	totalRetry := 1
+	if len(autoImportSecret.Annotations[constants.AnnotationAutoImportCurrentRetry]) != 0 {
+		lastRetry, err = strconv.Atoi(autoImportSecret.Annotations[constants.AnnotationAutoImportCurrentRetry])
+		if err != nil {
+			return reconcile.Result{}, err
+		}
 	}
 
-	// ensure the klusterlet manifest works exist
-	workSelector := labels.SelectorFromSet(map[string]string{constants.KlusterletWorksLabel: "true"})
```

Comment:
I think we need keep this check here to avoid apply the manifestwork early for backup-restore case